Package.swift

```diff
@@ -8,8 +8,7 @@ let package = Package(
         .package(url: "https://github.com/JohnSundell/Files.git", from: "2.0.0"),
         .package(url: "https://github.com/daniel-pedersen/SKQueue.git", from: "1.1.0"),
         .package(url: "https://github.com/surfandneptune/CommandCougar.git", from: "1.0.0"),
-//        .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
-        .package(url: "https://github.com/regexident/Rainbow", .branch("manifest")),
+        .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
```

Comment:
Should this be pinned to `from: 3.0.3` if this is the version that contains the necessary fix?

Reply:
@regexident? 😆

Reply:
`Package.resolved` pins it to `3.0.3` and `swift package resolve` should grab `3.0.3` over `3.0.0`, no?

Reply:
If `3.0.3` would be removed `swift package resolve` would use 3.0.2 instead of showing an error

Reply:
Fair point. Will fix in another PR then.